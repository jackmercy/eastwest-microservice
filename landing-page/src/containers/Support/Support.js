import { memo } from 'react';

import { SEO } from '../../components';
import { DEFAULT_SEO } from '../../../configs';
import { Layout, Header, Footer } from '../components';

const Support = () => (
  <Layout>
    <SEO {...DEFAULT_SEO.PAGES.SUPPORT} />
    <Header />
    <div>
      <h1>This is Support Page</h1>
      <p>Comming soon</p>
    </div>
    <Footer />
  </Layout>
);

export default memo(Support);
