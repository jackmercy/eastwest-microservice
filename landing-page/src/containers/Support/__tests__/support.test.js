import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import preloadAll from 'jest-next-dynamic';

import Support from '../Support';

describe('Testing Support', () => {
  beforeAll(async () => {
    // Header and Footer included in HomePage
    // And they are using dynamic import, see more in Header.js and Footer.js
    // So we need wait for it to load all before we can assert anything
    await preloadAll();
  });

  it('should render match snapshot', () => {
    const wrapper = shallow(<Support />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
