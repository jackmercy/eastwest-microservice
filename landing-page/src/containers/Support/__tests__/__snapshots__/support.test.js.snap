// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Support should render match snapshot 1`] = `
<Memo(Layout)>
  <SEO
    creator=""
    description="Description here"
    image=""
    keywords=""
    siteName=""
    title="East West - Support"
    url="https://dev.eastwest.live/support"
  />
  <ForwardRef(LoadableComponent) />
  <div>
    <h1>
      This is Support Page
    </h1>
    <p>
      Comming soon
    </p>
  </div>
  <ForwardRef(LoadableComponent) />
</Memo(Layout)>
`;
